src/interactive/versionslens.ts

```diff
@@ -0,0 +1,503 @@
+            ),
+            new vscode.CodeLens(
+                depsHeaderRange,
+                { title: 'Update all packages (Fixed)', command: updateAllDependenciesCommand, arguments: [UpgradeLevel.fixed] }
+            )
+        ]
+    }
+
+    /**
+     * See {@link vscode.HoverProvider}.
+     */
+    function provideFieldsAndHeadersHover(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken) {
+        const { name, uuid, version } = getProjectTomlFields(document)
+
+        if (uuid) {
+            const uuidRange = getFieldRange(document, 'uuid', uuid)
+            const hover = fieldHover('uuid', uuidRange, position)
+            if (hover) { return hover }
+        }
+
+        if (name) {
+            const nameRange = getFieldRange(document, 'name', name)
+            const hover = fieldHover('name', nameRange, position)
+            if (hover) { return hover }
+        }
+
+        if (version) {
+            const versionRange = getFieldRange(document, 'version', version)
+            const hover = fieldHover('version', versionRange, position)
+            if (hover) { return hover }
+        }
+
+        const sectionsHeadersRanges = getSectionsHeadersRanges(document)
+        for (const [sectionName, range] of sectionsHeadersRanges) {
+            if (range.contains(position)) {
+                return new vscode.Hover(
+                    Tooltips.sectionsHeaders[sectionName],
+                    range
+                )
+            }
+        }
+    }
+
+    function provideDepsFieldsHover(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken) {
+        const { deps } = getProjectTomlFields(document)
+
+        if (deps) {
+            const depsRanges = getSectionFieldsRanges(document, 'deps', deps)
+            return sectionHover('deps', depsRanges, position)
+        }
+    }
+
+
+    function provideExtrasFieldsHover(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken) {
+        const {  extras } = getProjectTomlFields(document)
+
+        if (extras) {
+            const extrasRanges = getSectionFieldsRanges(document, 'extras', extras)
+            return sectionHover('extras', extrasRanges, position)
+        }
+    }
+
+    function provideCompatFieldsHover(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken) {
+        const { compat } = getProjectTomlFields(document)
+
+        if (compat) {
+            const compatRanges = getSectionFieldsRanges(document, 'compat', compat)
+            return sectionHover('compat', compatRanges, position)
+        }
+    }
+
+    /**
+     * Execute `Pkg.update` and display its output while executing in an output channel.
+     */
+    async function _executeUpdateCommand(level: UpgradeLevel) {
+        const projectRoot = vscode.workspace.workspaceFolders[0]
```

Comment:
I would expect the update command to update the currently active project, not just what is in the root, right?